src/components/Message.vue

```diff
@@ -20,9 +20,11 @@
   -->
 
 <template>
-	<div :class="[message.hasHtmlBody ? 'mail-message-body mail-message-body-html' : 'mail-message-body']">
-		<div v-if="message.itineraries.length > 0" class="message-itinerary">
-			<Itinerary :entries="message.itineraries" :message-id="message.messageId" />
+	<div role="region" :aria-label="t('mail','Message body')">
```

Comment:
We don't need an extra div there? The role and aria-label can aimply be added to the existing div. :)